### PR TITLE
[GH-21] how to configure SSL in Portainer

### PR DESCRIPTION
**Pull request by 030**

### CODE REVIEW

File: docs/source/deployment.rst [hunk 1]
```diff
@@ -124,6 +124,27 @@ On Windows:
 
   $ docker run -d -p 9000:9000 -v C:\ProgramData\Portainer:C:\data portainer/portainer:windows
 
+SSL
```
Comment (deviantony) [inline docs/source/deployment.rst]: Please replace the SSL title with `Secure Portainer using SSL`.

File: docs/source/deployment.rst [hunk 2]
```diff
@@ -124,6 +124,27 @@ On Windows:
...
+===
+
+By default, Portainer runs without SSL. However it is possible to enable this, e.g.:
```
Comment (deviantony) [inline docs/source/deployment.rst]: Please replace *By default, Portainer runs without SSL. However it is possible to enable this, e.g.:* 

with 

*By default, Portainer's web interface and API is exposed over HTTP. This is not secured, it's recommended to enable SSL in a production environment. You can use the flags `--ssl`, `--sslcert` and `--sslkey` to enable SSL:*

File: docs/source/deployment.rst [hunk 3]
```diff
@@ -124,6 +124,27 @@ On Windows:
...
+
+This `document`_ instructs how to create a cert and a key:
+
+.. code-block:: bash
+
+  $ openssl genrsa -out server.key 2048
+  $ openssl ecparam -genkey -name secp384r1 -out server.key
+  $ openssl req -new -x509 -sha256 -key server.key -out server.crt -days 3650
+
+Note that `Certbot`_ could be used as well to create a cert and a key.
```
Comment (deviantony) [inline docs/source/deployment.rst]: Replace * to create a cert and a key.* with *to generate a certificate and a key.*

**Comment by 030** [comment]
@deviantony PR updated.

File: docs/source/deployment.rst [hunk 4]
```diff
@@ -124,6 +124,29 @@ On Windows:
 
   $ docker run -d -p 9000:9000 -v C:\ProgramData\Portainer:C:\data portainer/portainer:windows
 
+Secure Portainer using SSL
+==========================
+
+By default, Portainer's web interface and API is exposed over HTTP. This is not secured, it's recommended to enable SSL in a production environment. You can use the flags ``--ssl``, ``--sslcert`` and ``--sslkey`` to enable SSL:
+
+By default, Portainer runs without SSL. However it is possible to enable this, e.g.:
```
Comment (deviantony) [inline docs/source/deployment.rst]: You can remove this line.

**Comment by 030** [comment]
@deviantony I forgot to remove the line. Now it has been deleted.

**Comment by 030** [comment]
@deviantony Merged conflict solved after rebasing

**Comment by deviantony** [comment]
This might not be necessary after https://github.com/portainer/portainer/issues/821

**Comment by deviantony** [comment]
@030 looks like there is some kind of conflict with that PR, think you can fix it?

**Comment by deviantony** [comment]
Replaced by #23

**Pull request closed without merging**